WordGame/Scenes/Home/HomeViewController.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  HomeViewController.swift
+//  WordGame
+//
+//  Created by Kumari Ritu Pal on 25/07/22.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController, StoryboardHelper {
+    var interactor: HomeBuisnessLogic?
+    var viewModel: HomeViewModel?
+    
+    //MARK:- IBOutlets
+    @IBOutlet weak var lblCorrectAttempts: UILabel!
```

Comment:
If we are not accessing these outlets from outside of this ViewController, then we can make them private.
``` swift
@IBOutlet private weak var lblCorrectAttempts: UILabel!
```